### ISSUE TITLE
01513_optimize_aggregation_in_order_memory_long + randomized MergeTree settings = OOM

### ISSUE THREAD

**Issue by tavplubix** [ClickHouse/ClickHouse#46185]
https://play.clickhouse.com/play?user=play#d2l0aCAKJyUwMTUxM19vcHRpbWl6ZV9hZ2dyZWdhdGlvbl9pbl9vcmRlcl9tZW1vcnlfbG9uZyUnIGFzIG5hbWVfcGF0dGVybiwKJzIwMjItMDktMDEnIGFzIHN0YXJ0X2RhdGUsCignU3RhdGVsZXNzIHRlc3RzIChhc2FuKScsICdTdGF0ZWxlc3MgdGVzdHMgKGFkZHJlc3MpJywgJ1N0YXRlbGVzcyB0ZXN0cyAoYWRkcmVzcywgYWN0aW9ucyknKSBhcyBiYWNrcG9ydF9hbmRfcmVsZWFzZV9zcGVjaWZpY19jaGVja3MKc2VsZWN0IAp0b1N0YXJ0T2ZEYXkoY2hlY2tfc3RhcnRfdGltZSkgYXMgZCwKY291bnQoKSwgIGdyb3VwVW5pcUFycmF5KHB1bGxfcmVxdWVzdF9udW1iZXIpLCBhbnkocmVwb3J0X3VybCkKZnJvbSBjaGVja3Mgd2hlcmUgc3RhcnRfZGF0ZSA8PSBjaGVja19zdGFydF90aW1lIGFuZCBwdWxsX3JlcXVlc3RfbnVtYmVyIG5vdCBpbiAKKHNlbGVjdCBwdWxsX3JlcXVlc3RfbnVtYmVyIGFzIHBybiBmcm9tIGNoZWNrcyB3aGVyZSBwcm4hPTAgYW5kIHN0YXJ0X2RhdGUgPD0gY2hlY2tfc3RhcnRfdGltZSBhbmQgY2hlY2tfbmFtZSBpbiBiYWNrcG9ydF9hbmRfcmVsZWFzZV9zcGVjaWZpY19jaGVja3MpIAphbmQgdGVzdF9uYW1lIGxpa2UgbmFtZV9wYXR0ZXJuIAphbmQgdGVzdF9zdGF0dXMgaW4gKCdGQUlMJywgJ0ZMQUtZJykgZ3JvdXAgYnkgZCBvcmRlciBieSBkIGRlc2M=

Suppressed by https://github.com/ClickHouse/ClickHouse/pull/46180

**Comment by alexey-milovidov** [comment]
I see the recent commit from @Algunenano, fixing this test.

**Comment by Algunenano** [comment]
The commit I did was to support several errors from the server (for analyzer vs no analyzer) and in this specific test it was only removing some `;`. I didn't fix it; the original workaround (https://github.com/ClickHouse/ClickHouse/pull/46180) is still there.

**Comment by alexey-milovidov** [comment]
Random MergeTree settings are disabled for this test, and there was no investigation.
The original reports are outdated.

TLDR, we don't care about this.

**Comment by alexey-milovidov** [comment]
If you want, you can enable random MergeTree settings back and wait.
